scripts/run-mackoon-tests

```diff
@@ -83,7 +83,7 @@ fi
 
 # Run Mockoon container
 echo "Running mockoon container"
-RUNNING_ID=$("${CONTAINER_ENGINE}" run --name="${TEST_CONTAINER_NAME}" -d -v "${MOCK_JSON}":/data -p "${MOCKOON_PORT}":"${MOCKOON_PORT}" "${MOCKOON_CONTAINER}" -d /data)
+RUNNING_ID=$("${CONTAINER_ENGINE}" run --name="${TEST_CONTAINER_NAME}" -d -v "${MOCK_JSON}":/data --security-opt label=disable -p "${MOCKOON_PORT}":"${MOCKOON_PORT}" "${MOCKOON_CONTAINER}" -d /data)
```

Comment:
Can we use your `:Z` idea instead? It means we get to keep label enforcement. I just tried it on my fedora machine and it worked.

Reply:
yep there were 3 methods to fix it as described in the #430, changed to :Z